apps/web/src/app/(signing)/sign/[token]/form.tsx

```diff
@@ -103,6 +113,28 @@ export const SigningForm = ({ document, recipient, fields }: SigningFormProps) =
                 />
               </div>
 
+              <div>
```

Comment:
Could adjust this so that it only applies when the date field is in use?

Reply:
@dephraiim - Just pushed a commit for this to check if the date field is in the list of fields.